web/packages/teleport/src/Bots/Details/BotDetails.tsx

```diff
@@ -445,6 +445,8 @@ const traitDescriptions: { [key in (typeof traitsPreset)[number]]: string } = {
   host_user_gid: 'The group ID to use for auto-host-users',
   host_user_uid: 'The user ID to use for auto-host-users',
   github_orgs: 'List of allowed GitHub organizations for git command proxy',
+  mcp_tools: 'List of allowed MCP tools',
+  default_relay_addr: 'The relay address that clients should use by default',
```

Comment:
I added this description mostly to make `traitDescriptions` typecheck, but the `default_relay_addr` trait doesn't actually apply to bots. Should the description mention that, or should we expect to use these descriptions for trait descriptions for users too at some point?